File: otoko/trunk/Unix/trmode.f
C     LAST UPDATE 16/03/89
C+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
C
      SUBROUTINE TRMODE
      IMPLICIT NONE
C
C Purpose: Set terminal into transparent mode (alpha) 
C          after using graphics.
C          n.b.  This routine does not clear any screens.
C
C Calls   0:
C
C-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-
C LOCAL VARIABLES:
C
      CHARACTER*1 CAN
C
      DATA CAN/24/
C
C-----------------------------------------------------------------------
C
      call flush (6)
      PRINT 1000,CAN
      RETURN
C
1000  FORMAT (A1)
      END

File: otoko/Unix/trmode.c
/***********************************************************************
 *                          trmode.c                                   *
 ***********************************************************************
 Purpose: Set terminal into transparent mode (alpha) 
          after using graphics.
          n.b.  This routine does not clear any screens.
 Authors: G.R.Mant
 Returns: Void
 Updates: 
 23/07/92 GRM Initial implementation
*/

#include <stdio.h>

#ifdef titan                            /* Stardent */
#define send_can TRMODE
#else
#ifdef hpux                    /* brain dead hp */
#define send_can trmode
#else
#define send_can trmode_                /* normal computers */
#endif
#endif

#define CAN '\030'

void send_can ()
{

      fflush (stdout);
      putc (CAN, stdout);

}
